go/vt/vttablet/tabletserver/tabletenv/tabletenv.go

```diff
@@ -65,27 +67,48 @@ var (
 		vtrpcpb.Code_DATA_LOSS.String(),
 	)
 	// InternalErrors shows number of errors from internal components.
-	InternalErrors = stats.NewCounters("InternalErrors", "Task", "StrayTransactions", "Panic", "HungQuery", "Schema", "TwopcCommit", "TwopcResurrection", "WatchdogFail", "Messages")
+	InternalErrors = stats.NewCountersWithLabels("InternalErrors", "Number of errors from internal components", "component", "Task", "StrayTransactions", "Panic", "HungQuery", "Schema", "TwopcCommit", "TwopcResurrection", "WatchdogFail", "Messages")
 	// Warnings shows number of warnings
-	Warnings = stats.NewCounters("Warnings", "ResultsExceeded")
+	Warnings = stats.NewCountersWithLabels("Warnings", "Number of warnings", "type", "ResultsExceeded")
 	// Unresolved tracks unresolved items. For now it's just Prepares.
-	Unresolved = stats.NewCounters("Unresolved", "Prepares")
+	Unresolved = stats.NewGaugesWithLabels("Unresolved", "Tablet env's unresolved items", "item_type", "Prepares")
```

Comment:
"Tablet env" is an internal abstraction. I'd just describe this as "Unresolved items"